### ISSUE TITLE
No way to close the color picker

### ISSUE THREAD

**Issue by yourAverageDeveloper** [narsenico/a-color-picker#13]
Hey! Thanks for the awesome color picker!
But there's no default way to close it. I did get it working by manually adding a display property but it would be great if such a functionality comes out of the box

**Comment by mhebrard** [comment]
Facing the same issue here. I like the style of the color picker :) but I am a bit annoyed that there is no close button. 
@yourAverageDeveloper May I know to which event did you bind the closing ? or where did you integrate the button ?

**Comment by jclaessens97** [comment]
Same here.

**Comment by yourAverageDeveloper** [comment]
> Facing the same issue here. I like the style of the color picker :) but I am a bit annoyed that there is no close button.
> @yourAverageDeveloper May I know to which event did you bind the closing ? or where did you integrate the button ?

What i did was very simple really. I just added a button in the div where the picker was gonna be

`<div class='picker'><button class='close-picker'>Close</button</div>`

I added some CSS to the button positioning it absolutely while positioning the picker relatively. And then just moved it some negative pixels left and up. And then just added an event listener to the button

`$('.close-picker').addEventListener('click', () =>
{
  picker.css('display', 'none')
}`
